C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -100,7 +108,20 @@ public async Task<SlackAPI> GetAPI(Activity activity)
         /// <returns></returns>
         public async Task<string> GetBotUserByTeam(Activity activity)
         {
-            return "";
+            if (!string.IsNullOrEmpty(Identity))
+            {
+                return Identity;
+            }
+            else if ((activity.Conversation as dynamic).team != null)
+            {
+                var userID = await options.GetBotUserByTeam((activity.Conversation as dynamic).team);
+                return string.IsNullOrEmpty(userID) ? userID : throw new Exception("Missing credentials for team.");
```

Comment:
This should be 
```
return string.IsNullOrEmpty(userID) ? throw new Exception("Missing credentials for team.") : userID;
```

Reply:
This is still the same.

Reply:
I'm seeing this in the code
`return !string.IsNullOrEmpty(userID) ? userID : throw new Exception("Missing credentials for team.");`